src/processevent.cpp

```diff
@@ -18,7 +18,8 @@ namespace ProcessEvent
         QJsonDocument jsonDoc;
         if (!message.startsWith("{"))
         {
-            auto str = QString("{\"idx\": %1, \"k\": \"\", \"v\": \"%2\"}").arg(QString::number(index), message);
+            auto str = QString("{\"idx\": %1, \"k\": \"\", \"v\": \"%2\"}")
+                    .arg(QString::number(index), message.replace("\\", "\\\\"));
```

Comment:
what kind of events get a better display? does this only apply to plain text files?